documentation/badge/badge.css

```diff
@@ -0,0 +1,148 @@
+@import "/documentation/Styles/general.css";
+
+@media screen and (max-width: 824px) {
+    html {
+        font-size: 14px;
+    }
+
+    .side-bar {
+        margin-top: 40px;
+    }
+
+    .body-container {
+        margin-top: 60px;
+    }
+}
+
+@media screen and (max-width: 715px) {
+    html{
+        font-size: 11px;
+    }
+    .code-container{
+        display: none;
+    }
+    .footer-btn {
+        font-size: 1.4rem;
+    }
+
+}
+
+.side-badge-btn {
+    background-color: var(--header);
+    border-radius: 25px;
```

Comment:
maybe try using rems for the border radius

Reply:
I will take care in the future.